neuralprocesses/architectures/convgnp.py

```diff
@@ -22,6 +22,8 @@ def construct_convgnp(
         unet_channels=(64,) * 6,
         unet_activations=None,
         unet_kernels=5,
+        unet_use_resize_convs=True,
```

Comment:
Perhaps we want to call `unet_use_resize_convs` instead `unet_resize_convs` so that it is a prefix of `unet_resize_conv_interp_method`? That looks nice aesthetically. :)

Reply:
Great suggestion! I'll implement in next push.

Reply:
I will also update the `UNet` arg from `use_resize_convs` to `resize_convs` to match up.